system-tests/common.py

```diff
@@ -352,9 +344,6 @@ class CompareConsoleWithFile(CheckTask):
     """
     # The expected result is held in a file "expected.<tag>[.<suffix>].txt.
     # Where "tag" comes from the task and "suffix" is specified.
-    #
-    # This comment is here because systest highjacks the doc string for output.
-    #
```

Comment:
The point being that the comment can now once again go in the doc-string where it was supposed to be.

Reply:
Ah right!  That makes sense, for some reason I thought the "This comment is here" only referred to itself and not the comment above, and I didn't spend long thinking about what it actually meant